src/views/v2/components/events-bar/views/list/item.php

```diff
@@ -27,7 +27,9 @@
 		class="tribe-events-c-view-selector__list-item-link"
 		data-js="tribe-events-view-link"
 	>
-		<span class="tribe-events-c-view-selector__list-item-icon tribe-common-svgicon tribe-common-svgicon--<?php echo esc_attr( $public_view_slug ); ?>"></span>
+		<span class="tribe-events-c-view-selector__list-item-icon">
+			<?php $this->template('components/icons/' . esc_attr( $public_view_slug ), [ 'classes' => ['tribe-events-c-view-selector__list-item-icon-svg'] ] ); ?>
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space after array opener.
* :no_entry_sign: Missing space before array closer.
* :no_entry_sign: Missing space before array closer.

<sub>hash: 20ec2f11bde12c0a528b5e61d975931f</sub>